lib/signed-xml.js

```diff
@@ -316,6 +313,9 @@ function SignedXml(idMode, options) {
   this.idAttributes = ["Id", "ID", "id"];
   if (this.options.idAttribute) this.idAttributes.splice(0, 0, this.options.idAttribute);
   this.implicitTransforms = this.options.implicitTransforms || [];
+  this.composeKeyInfoContent = SignedXml.composeKeyInfoContent;
```

Comment:
I'd like to share a perspective from a non-native English speakers viewpoint on this decision. The term "compose" seems slightly more unfamiliar to me. I believe the earlier choice of `getKeyInfoContent` is somewhat more conventional and concise.